src/inputs/DateField.mock.test.tsx

```diff
@@ -35,10 +35,13 @@ describe("MockDateField", () => {
   });
 
   it("forwards disabledDays prop as stringified object", async () => {
-    const testDate = new Date(2020, 0, 1)
-    const r = await render(<MockDateField label="Start Date" value={testDate} disabledDays={{ before: testDate }} onChange={() => {}} />);
+    const testDate = new Date(2020, 0, 1);
+    const disabledDays = [{ before: testDate }, () => false];
```

Comment:
I'm not sure I understand this noop `() => false` as a `Modifier`? it that some special case?

Reply:
It's to represent a FunctionModifier, which I'm using to [disable days for the scheduleSettings](https://github.com/homebound-team/internal-frontend/pull/1794/files#diff-b5da248aa7ad502a7ca5b33777e5a23e60a778ec6084a912bf80471f381480e1R61).